workflow/_circles.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import division
+
```

Comment:
very minor: since those two files are only used by the mask score, I would move them into /scores/

Reply:
Do you consider `iou.py` being part of the scores too ?

Reply:
Maybe not, as we might want to expose that to participants? (so they can import and use that in their code ?)